src/graphql/admin/index.ts

```diff
@@ -9,17 +9,26 @@ import { ALL_INTERFACE_TYPES } from "./types"
 export { queryFields as adminQueryFields } from "./queries"
 export { mutationFields as adminMutationFields } from "./mutations"
 
-if (isDev && !isRunningJest) {
-  import("@services/fs").then(({ writeSDLFile }) => {
-    writeSDLFile(
-      __dirname + "/schema.graphql",
-      printSchema(lexicographicSortSchema(gqlAdminSchema)),
-    )
-  })
-}
-
 export const gqlAdminSchema = new GraphQLSchema({
   query: QueryType,
   mutation: MutationType,
   types: ALL_INTERFACE_TYPES,
 })
+
+const updateSchemaGraphql = async () => {
+  console.log("updating schema.graphql")
```

Comment:
same here

Reply:
no need. it's just used as output info when running github action/locally (it's a script) so no need to wrap it in object